lib/internal/repl/history.js

```diff
@@ -97,7 +97,7 @@ function setupHistory(repl, historyPath, ready) {
     }
 
     if (data) {
-      repl.history = RegExpPrototypeSymbolSplit(/[\n\r]+/, data, repl.historySize);
+      repl.history = RegExpPrototypeSymbolSplit(/\n+/, data, repl.historySize);
```

Comment:
is the `\r` not needed here for windows users?

Reply:
I talked with @puskin94 about it and we define what we write into the file. That is why it's fine to keep it as `\n`.

Reply:
Seems like that is not true though, or it fails as a different point.
Windows tests are all failing.

Reply:
Investigating. When I did my manual tests on windows I didn't see any issue. Or something broke with the most recent commits or I didn't manually test some specific cases covered by the automated tests

Reply:
it turns out it was either of those 😄 
I never changed the `flushHistory` function in the history file, that means it always joined the history lines before saving them in the history file with `os.EOL`, which is `\n\r` on windows, but `\n` anywhere else.
Normalized it to be just `\n`